Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM golang:1.17.3
+
+# Install packages
+RUN curl -sL https://deb.nodesource.com/setup_14.x | bash -
```

Comment:
Does this need to be version 14? I think 16 is the "active" version.

Reply:
What is this used for? I don't remember seeing NodeSource before.

Reply:
That's the setup option that uses Ubuntu's Personal Package Archive.